### ISSUE TITLE
Visual Studio code throwing exception when creating Azure Logic apps(Preview) Project

### ISSUE THREAD

**Issue by amarmanchalagithub** [Azure/logicapps#189]
Does this occur consistently? 
Repro steps:

1.
2.

Action: azureLogicAppsV2.createNewProject
Error type: Error
Error Message: Failed to get latest templates: Unexpected token E in JSON at position 0

Version: 0.0.4
OS: win32
OS Release: 6.1.7601
Product: Visual Studio Code
Product Version: 1.50.1
Language: en

<details>
<summary>Call Stack</summary>

```
CentralTemplateProvider.<anonymous> extension.bundle.js:164:430605
fulfilled extension.bundle.js:164:427159
processTicksAndRejections task_queues.js:94:5
```

</details>

**Comment by vikash-anand** [comment]
Hi @amarmanchalagithub This seems like an issue with installation. Can you please go to **C:\Users<Username>.vscode\extensions** and delete all **ms-azuretools.vscode-azurelogicapps-*** and then try to install the extension again?

**Comment by amarmanchalagithub** [comment]
i tried it..still running into the same problem

**Comment by mcollier** [comment]
I'm running into this problem too.

Action: azureLogicAppsV2.createNewProject
Error type: Error
Error Message: Failed to get latest templates: Unexpected token E in JSON at position 0

Version: 0.0.5
OS: win32
OS Release: 10.0.19042
Product: Visual Studio Code
Product Version: 1.50.1
Language: en

<details>
<summary>Call Stack</summary>

```
CentralTemplateProvider.<anonymous> extension.bundle.js:164:430605
fulfilled extension.bundle.js:164:427159
processTicksAndRejections task_queues.js:94:5
```

</details>

**Comment by mcollier** [comment]
Update - this occurs when trying to create a new Logic App (Preview) from the VS Code extension.

![VS Code - Logic App Preview error](https://user-images.githubusercontent.com/1142273/98262738-f5b5a180-1f53-11eb-8698-9a9c408f1df8.gif)

**Comment by vikash-anand** [comment]
@mcollier @amarmanchalagithub We have released a new version of the extension - v0.0.6 today which should fix this issue. Please update your extension and let us know if the issue exists.

**Comment by mcollier** [comment]
I just tried with v0.0.6, and it works!  I was able to retrieve the templates and create a new Stateless project.
 
![image](https://user-images.githubusercontent.com/1142273/98963314-7be26280-24d5-11eb-938d-a7df2f8a2b20.png)
